ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/graphSearch/stepExpansion/ParameterBasedStepExpansion.java

```diff
@@ -245,4 +247,12 @@ private DiscreteFootstep constructNodeInPreviousNodeFrame(double stepLength, dou
                                   stepYaw + step.getYaw(),
                                   step.getRobotSide().getOppositeSide());
    }
+
+   /**
+    * This was made to use in tests to check if the number of yaw offsets is what we expect
+    */
+   public TDoubleArrayList getYawOffsets()
```

Comment:
could you make it package private? this really probably shouldn't be used by a user. 

You could also make the argument that this should be done by packing an argument, rather than returning a value like this to avoid mutability. 

`void getYawOffsets(TDoubleArrayList yawOffsetsToPack)`

That makes sure no one is modifying it.

Reply:
Ya that is good, I like that